### PR TITLE
Ttucker tap

### PR DESCRIPTION
**Pull request by tylertucker202**
Replacing metadata keyword tables with TAP tables.

### CODE REVIEW

File: create_ipac_tables/README [hunk 1]
```diff
@@ -0,0 +1,15 @@
+# koa_tap_table_to_ipac_table.py
```
Comment (joshjriley) [inline create_ipac_tables/README:1]: This folder looks like some test/validation code i think.  I'm wondering if we should organize this into a general test folder.  This being the first test and hopefully we will add more later on.

File: dep_dqa.py [hunk 2]
```diff
@@ -164,7 +164,7 @@ def dep_dqa(instrObj, tpx=0):
     tablesDir = instrObj.metadataTablesDir
     ymd = utDate.replace('-', '')
     metaOutFile =  dirs['lev0'] + '/' + ymd + '.metadata.table'
-    keywordsDefFile = tablesDir + '/keywords.format.' + instr
+    keywordsDefFile = tablesDir + f'/KOA_{instr.upper()}_Keyword_Table.txt'
```
Comment (joshjriley) [inline dep_dqa.py:167]: Have we decided on the checkout location for the KeywordTables repo?  I'm thinking vm-koaserver5:/kroot/archive/KeywordTables/

File: metadata.py [hunk 3]
```diff
@@ -16,15 +16,17 @@
...
-from common import make_dir_md5_table
 import datetime
 import re
 import pandas as pd
 import html
+import pdb
+import glob
+import gzip
+import hashlib
+import logging
```
Comment (joshjriley) [inline metadata.py:27]: From my test, the logging in this file does not get added to the main output log file that is defined in creae_log.py.  I did a quick test, and I think we can solve this by getting the logging object by the same name as it was generated in create_log.py.  So I put this at the top of metadata.py in the module global scope and changed the "logging." calls to "log.":

`
log = logging.getLogger(f"dep <{os.getlogin()}>")
`

**Review by joshjriley — CHANGES_REQUESTED** [review]
Just need to fix the file logging.

File: create_ipac_tables/README (hunk 1 above)
Comment (tylertucker202) [reply create_ipac_tables/README:1]: Get out of my head. I was thinking the same thing. I will do some refactoring to make this more test-like.

File: dep_dqa.py (hunk 2 above)
Comment (tylertucker202) [reply dep_dqa.py:167]: We have not. I will try putting them there.

File: metadata.py (hunk 3 above)
Comment (tylertucker202) [reply metadata.py:27]: This will be addressed and tested today. Thanks for spotting it.

File: dep_dqa.py (hunk 2 above)
Comment (tylertucker202) [reply dep_dqa.py:167]: No go. Perhaps someone else has writing privileges to /kroot/archive?

**Comment by tylertucker202** [comment]
Comments have been addressed as follows:
- new file test contains metadata.py test. 
- test suite uses pytest to generate test on small dataset
- I created a toy dataset from data taken from koa. I can transfer if needed. Otherwise, any koaarchive can be used. The tests do assume that each instrument is included. 
- script includes logging test
-

**Pull request merged**